fast_depends/use.py

```diff
@@ -48,7 +48,7 @@ def __call__(
 
 @overload
 def inject(  # pragma: no cover
-    func: None,
+    func: None = None,
```

Comment:
I also think it would also be reasonable to just do:
```suggestion
```
but this way you at least have the option to purposely use `func=None` as an argument if desirable for some exotic reason..

Reply:
Yes, indeed, but I think, that it should be `Literal[None] = None`